cpp/src_prims/sparse/selection/knn.cuh

```diff
@@ -498,13 +443,14 @@ void brute_force_knn(const value_idx *idxIndptr, const value_idx *idxIndices,
                      cudaStream_t stream,
                      size_t batch_size_index = 2 << 14,  // approx 1M
                      size_t batch_size_query = 2 << 14,
-                     ML::MetricType metric = ML::MetricType::METRIC_L2,
-                     float metricArg = 0, bool expanded_form = false) {
+                     raft::distance::DistanceType metric =
+                       raft::distance::DistanceType::L2Unexpanded,
```

Comment:
Same, `L2SqrtUnexpanded` might be needed here.

Reply:
And here.